starter-code/index.html

```diff
@@ -10,6 +10,17 @@
     <link rel="stylesheet" href="styles/layout.css">
     <link rel="stylesheet" href="styles/modules.css">
     <link rel="icon" type="image/x-icon" href="https://avatars2.githubusercontent.com/u/18607604?v=3&s=200">
+    <script id="article-template" type="text/x-handlebars-template">
+      <article class="template" data-category="cat">
+        <header>
+          <h1>{{title}}</h1>
+          <div class="byline">
+            By <address><a href="">{{author}}</a></address>
+            published <time pubdate datetime="2000-01-01">{{publishedOn}}</time>
```

Comment:
`published <time pubdate datetime="{{publishedOn}}">{{publishedStatus}}</time>`
you are just missing the handlebars expression for datetime and publishedOn should be publishedStatus (take a look at article.js line 26)